### PR TITLE
Feature/spa rw

### PR DESCRIPTION
**Pull request by rachelmariewhite**
Header and SPA

### CODE REVIEW

**Comment by romulocintra** [comment]
This PR does not anwser the task

File: public/src/videos-page.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class VideosPage extends HTMLElement {
+    constructor() {
+        super();
+    }
+    connectedCallback() {
+        this.innerHTML = `Video Test`
```
Comment (romulocintra) [inline public/src/videos-page.js]: This is not previous HTML that we made on Sessions

**Review by romulocintra — CHANGES_REQUESTED** [review]

File: public/src/videos-page.js (hunk 1 above)
Comment (rachelmariewhite) [reply public/src/videos-page.js]: not sure I follow, but added the HTML in the `video test` section and did another pull request.

**Comment by rachelmariewhite** [comment]
corrected header, footer and video-page.js

**Comment by rachelmariewhite** [comment]
add new header, footer and HTML in video-page.js

**Pull request merged**